model_app/.gitignore

```diff
@@ -1,3 +1,6 @@
 /node_modules/*
 /frontend/node_modules/*
 /backend/node_modules/*node_modules/
+node_modules/
```

Comment:
Should node_modules/ be repeated 3x here? The most recent commit on master takes care of all of this because the model_app, frontend, and backend all have .gitignores which cover the 3 node_modules + 3 package-lock.jsons